### ISSUE TITLE
get  loss_box =0

### ISSUE THREAD

**Issue by zqdeepbluesky** [bosch-ros-pkg/bstld#7]
Hi, When I trained the bosch traffic light dataset with faster rcnn, I got loss_box=0. Does it mean that the label box of the object in the dataset is too small for faster rcnn to detect?

**Comment by karstenBehrendt** [comment]
Hi @zqdeepbluesky !
Unfortunately, this question does not provide enough insights to give a useful response. There are several Faster RCNN approaches that work nicely with the dataset. I don't know which framework you are using or even what the loss_box is within this context.

The issue section here is about the code of this repo and issues with the annotations. For hints on how to train a neural network or issues with frameworks, I recommend stackoverflow or the framework itself.

I will close as off-topic.
